src/app/shared/filter-bar/label-filter-bar/label-filter-bar.component.ts

```diff
@@ -93,4 +94,8 @@ export class LabelFilterBarComponent implements OnInit, OnDestroy {
   filter(filter: string, target: string): boolean {
     return !target.toLowerCase().includes(filter.toLowerCase());
   }
+
+  removeAllSelection(): void {
+    this.matSelectionList.deselectAll();
```

Comment:
There's a problem with this actually, it doesn't update the label filter on the issues, i.e. selectionChange doesn't run

After updating the view, call what's inside selectionChange on mat-selection-list, to run the behaviour of manually clicking the checkbox

Reply:
Don't forget to abstract that into another method so that there is no duplicated code